tests/es6-super.js

```diff
@@ -48,26 +48,26 @@ describe("ES6 super", () => {
 
         expect(scope.type).to.be.equal("global");
         expect(scope.variables).to.have.length(1);
-        expect(scope.variables[0].name).to.be.equal("Hello");
-        expect(scope.references).to.have.length(0);
+        expect(scope.variables[0].name).to.be.equal("Foo");
+        expect(scope.references).to.have.length(1);
```

Comment:
The newly created reference here is a reference to `Bar` - do we want to add some assertions around that?